src/Equations/SlowingDownFrequency.cpp

```diff
@@ -124,20 +135,26 @@ real_t SlowingDownFrequency::GetAtomicParameter(len_t iz, len_t Z0){
     }
     if (Z <= MAX_Z){ /* use tabulated data */
         I = MEAN_EXCITATION_ENERGY_DATA[Z-1][Z0];
-    }else{ /* use the formula instead */
-        len_t Ne = Z-Z0;
-        if (Ne <= MAX_NE){
-            D_N = MEAN_EXCITATION_ENERGY_FUNCTION_D[Ne-1]; 
-            S_N0 = MEAN_EXCITATION_ENERGY_FUNCTION_S_0[Ne-1];
-        }else{
-            D_N = MEAN_EXCITATION_ENERGY_FUNCTION_D[MAX_NE-1]; 
-            S_N0 = Ne - sqrt(Ne*HIGH_Z_EXCITATION_ENERGY_PER_Z / HYDROGEN_MEAN_EXCITATION_ENERGY); // S_N0: for a neutral atom with Z=N
-        }
-        real_t A_N = (1-D_N) * (1-D_N);
-        real_t B_N = 2*(1-D_N) * (Ne*D_N - S_N0);
-        real_t C_N = (Ne*D_N - S_N0) * (Ne*D_N - S_N0);
-
-        I = HYDROGEN_MEAN_EXCITATION_ENERGY * (A_N*Z*Z + B_N*Z + C_N);
+    }
+    else    { /* use extended data from MeanExcitationEnergy_Extended.cpp */
+		if (Z <= MAX_Z_EXTENDED){ /*this if can be used to switch to the old version of code*/
```

Comment:
This could be written as
```c++
else if (Z <= MAX_Z_EXTENDED) {
```
instead, to avoid unnecessary levels of nested if's.